src/controllers/still.required.controller.ts

```diff
@@ -8,7 +8,7 @@ import {getPromiseToFileSessionValue} from "../services/session.service";
 import {COMPANY_PROFILE} from "../session/keys";
 
 const validators = [
-  check("continuedIllness").not().isEmpty().withMessage(COMPANY_REQUIRED_NOT_SELECTED),
+  check("stillRequired").not().isEmpty().withMessage(COMPANY_REQUIRED_NOT_SELECTED),
```

Comment:
This has been done on ticket LFA-1380

Reply:
Yes but this validation error is holding up the ticket that this bug corresponds to.